libs/ledger-live-common/src/api/Ethereum.ts

```diff
@@ -116,9 +116,10 @@ export type BlockByHashOutput = {
 export type API = {
   getTransactions: (
     address: string,
-    blockHeight: number | null | undefined,
-    batchSize?: number
-  ) => Promise<Tx[]>;
+    blockHeight?: number | null,
```

Comment:
This is in the function input not in an object, this allows the function to be called without the input, it's different imo

Reply:
The arguments of a function is technically an array/object as well:
```typescript
function hello (a,b,c) {
    return arguments.length;
}
hello("a", "b"); // 2
hello("a", "b", undefined); // 3
```
So here if you were rellying on `arguments.length` you can have the same kind of "side effects" by implying that `anything?: string | null | undefined` is the same thing as `anything?: string | null` because the value is optional.

I'm just picky here because I know it's been a topic lately that we should remove all the `undefined` in those cases in the project in general, but in some cases, it creates some unwanted behaviours.

Reply:
Oh ok just overall remark then, here we don't use the args array

It's the same topic as not defining a key in an object vs setting the value as undefined